src/main/kotlin/com/manimdsl/frontend/AbstractSyntaxTree.kt

```diff
@@ -1,7 +1,19 @@
 package com.manimdsl.frontend
 
+import javax.swing.plaf.nimbus.State
```

Comment:
Is this used?

Reply:
sorry this is not meant to be there I thought I deleted it